core/src/main/antlr4/io/delta/sql/parser/DeltaSqlBase.g4

```diff
@@ -87,8 +87,11 @@ statement
     | ALTER TABLE table=qualifiedName
         DROP CONSTRAINT (IF EXISTS)? name=identifier                    #dropTableConstraint
     | OPTIMIZE (path=STRING | table=qualifiedName)
-        (WHERE partitionPredicate = predicateToken)?
+        (WHERE partitionPredicate=predicateToken)?
         (zorderSpec)?                                                   #optimizeTable
+    | REORG TABLE table=qualifiedName
```

Comment:
what about `path`? If it's legal for OPTIMIZE, why not legal for REORG TABLE?

Reply:
Talked with Bart. He believes that the string literal version of OPTIMIZE is nonstandard, and is not something we would not recommend to users.